documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -60,8 +60,12 @@ spec:
 |`10`
 
 |`controller_filesystem_overhead`
-|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this label can only be changed using the CLI
+|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this label can only be changed using the CLI.
 |`10`
-|===
 
+controller_block_overhead
 
+|`controller_block_overhead`
+|Space in persistent volumes allocated as block overhead when the `storageclass` is block-based. Note, this label can only be changed using the CLI.
```

Comment:
Space sounds a bit too wide for me. Does it sounds like it's a fixed number? Also, user may use it as resource (e.g 1G, 1M..)

Reply:
Tweaked slightly.

Reply:
> Tweaked slightly.

looks great now